conslogging/conslogging.go

```diff
@@ -130,17 +130,35 @@ func (cl ConsoleLogger) WithFailed(isFailed bool) ConsoleLogger {
 }
 
 // PrintSuccess prints the success message.
-func (cl ConsoleLogger) PrintSuccess() {
+func (cl ConsoleLogger) PrintSuccess(extra string) {
 	cl.mu.Lock()
 	defer cl.mu.Unlock()
-	cl.color(successColor).Fprintf(cl.outW, "=========================== SUCCESS ===========================\n")
+	cl.printBar(" SUCCESS ", extra)
 }
 
 // PrintFailure prints the failure message.
-func (cl ConsoleLogger) PrintFailure() {
+func (cl ConsoleLogger) PrintFailure(extra string) {
 	cl.mu.Lock()
 	defer cl.mu.Unlock()
-	cl.color(warnColor).Fprintf(cl.outW, "=========================== FAILURE ===========================\n")
+
+	cl.printBar(" FAILURE ", extra)
```

Comment:
Wouldn't this be printed with the incorrect color?

Reply:
Shoot, victim of copy/paste